connector_jira/models/account_analytic_line/importer.py

```diff
@@ -49,6 +49,21 @@ def issue(self, record):
             refs['jira_epic_issue_key'] = issue['fields'][epic_field_name]
         return refs
 
+    @mapping
+    def date(self, record):
+        mode = self.backend_record.worklog_date_timezone_mode
+        started = record['started']
+        _logger.info('!!!! %s %s' % (mode, started))
```

Comment:
is it needed to log here? especially as INFO?

Reply:
Sorry, leftover, removed